benches/end2end.rs

```diff
@@ -60,17 +60,17 @@ fn end2end_benchmark(c: &mut Criterion) {
     let reduction_count = DEFAULT_REDUCTION_COUNT;
 
     // setup
-    let lang_pallas = Lang::<Fq, Coproc<Fq>>::new();
-    let lang_pallas_rc = Arc::new(lang_pallas.clone());
+    let lang = Lang::<Fq, Coproc<Fq>>::new();
+    let lang_arc = Arc::new(lang);
```

Comment:
Have a look at how Rc and Arc are used here: https://github.com/bodil/im-rs
This is actually a frequent pattern

Reply:
Okay, I can change it back to _rc. But most of these lines would have to be changed anyway because the API changed.

Please let me know if you want me to proceed.
Note: now that we have these changed already, It's mostly unnecessary effort at this point. So I'd rather move on and avoid the same issue in the future

Reply:
> Note: now that we have these changed already, it's mostly unnecessary effort at this point. So I'd rather move on and avoid the same issue in the future

Agreed